sqlglot/dialects/risingwave.py

```diff
@@ -1,6 +1,501 @@
+from __future__ import annotations
 from sqlglot.dialects.postgres import Postgres
+from sqlglot.tokens import TokenType
+from collections import defaultdict
+import typing as t
+
+from sqlglot import exp
 
 
 class RisingWave(Postgres):
+    class Tokenizer(Postgres.Tokenizer):
+        KEYWORDS = {
+            **Postgres.Tokenizer.KEYWORDS,
+            "SINK": TokenType.SINK,
+            "SOURCE": TokenType.SOURCE,
+        }
+
+    class Parser(Postgres.Parser):
+        DB_CREATABLES = {
+            *Postgres.Parser.DB_CREATABLES,
+            TokenType.SINK,
+            TokenType.SOURCE,
+        }
+        CREATABLES = {
+            *Postgres.Parser.CREATABLES,
+            *DB_CREATABLES,
+        }
+
+        ID_VAR_TOKENS = {
+            *Postgres.Parser.ID_VAR_TOKENS,
+            *CREATABLES,
+        }
+
```

Comment:
Let's delete these and add `TokenType.SINK` and `TokenType.SOURCE` in the base `DB_CREATABLES` set.